src/selectors/layout.test.js

```diff
@@ -23,7 +23,8 @@ describe('Selectors', () => {
     const prepareLargeDataset = () => {
       const data = { ...animals };
       let extraNodes = [];
-      const iterations = Math.ceil(largeGraphThreshold / data.nodes.length) + 1;
+      const iterations =
```

Comment:
Please add a comment here to mention that we need to increase the `largeGraphThreshold` amount by double the amount here to ensure the warning gets triggered in the tests given parameter nodes are now turned off.